frontend/admin/src/js/components/dashboard/Dashboard.tsx

```diff
@@ -8,11 +10,9 @@ import Header from "@common/components/Header";
 import Footer from "@common/components/Footer";
 import useIsSmallScreen from "@common/hooks/useIsSmallScreen";
 import { SideMenuContentWrapper } from "@common/components/SideMenu";
-
-import { Bars3Icon } from "@heroicons/react/24/outline";
 import NotAuthorized from "@common/components/NotAuthorized";
+
 import AdminSideMenu from "../menu/AdminSideMenu";
-import { ADMIN_APP_DIR } from "../../adminConstants";
```

Comment:
Ahhh, I did not realize they were using two different folders. Maybe we should look into a shared assets folder to avoid duplicating images?

Fixed in [add logos to admin](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4049/commits/af2cd58d00f5c1798988fd96be243e9c6e24b891)

Reply:
No objection, but wondering if we couldn't just prioritize the issue to merge apps? https://github.com/GCTC-NTGC/gc-digital-talent/issues/2917

I'm sure there's a reason, but I'd also love to start using regular image imports and let webpack handle the images, instead of assembling urls like this. That way, broken image links are a non-issue, because that's a compilation error.

Reply:
Yeah @patcon , I agree with you on this one. I would like to get the merge in before we begin rebranding so we can share assets.

Reply:
Updated this so we use components in the `common` project for logos. This also means we no longer need to pass the `baseUrl` prop to `Header` and `Footer`.

[move logos to components](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4049/commits/aff87d0828265e5ec7ab735db855d7cc0034fc1c)

Reply:
Yeah, I'd be very happy prioritizing #2917! I think it will have to wait until after the Direct Intake release, but I'll move it to the top of the Safebox column.